### PR TITLE
Updated downgrade steps for OCP 3.4

### PR DESCRIPTION
**Pull request by ahardin-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1457114

This addresses the downgrade process from OCP 3.4 to 3.3.

Preview Build: http://file.rdu.redhat.com/~ahardin/06212017/update-34-downgrade/install_config/downgrade.html

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@adellape @mtnbikenc PTAL

**Comment by adellape** [comment]
See comments from the 3.5 version of this PR for the similar parts: https://github.com/openshift/openshift-docs/pull/4570#pullrequestreview-46628120

File: install_config/downgrade.adoc [hunk 1]
```diff
@@ -173,49 +156,46 @@ Delete all images:
...
-    --enable=rhel-7-server-ose-3.2-rpms
+    --disable=rhel-7-server-ose-3.4-rpms \
+    --enable=rhel-7-server-ose-3.3-rpms
 ----
-====
 
-On each master, install the following packages:
+. On each master, install the following packages:
 
-====
```
Comment (adellape) [inline install_config/downgrade.adoc]: Also here is missing a linking `+`.

**Comment by ahardin-rh** [comment]
@adellape Thanks! Updated.

**Comment by ahardin-rh** [comment]
Verified by QE

**Pull request merged**